RPIPs/RPIP-34.md

```diff
@@ -0,0 +1,365 @@
+
+**Transparency**
+
+The outcomes of processes can be recorded in a clear, consistent, and accessible manner. This is worthwhile in and of itself, but it also reduces the attention and mental effort required to find relevant information.
+
+**Extensibility**
+
+Adoption of an operational process framework helps to structure the processes of the Rocket Pool DAOs in such a way that they are easily extensible in future improvement proposals. The motivation here is to encourage organization, transparency, and accessibility in such a way that it becomes self-sustaining. If there is a clear existing structure in use, then it encourages its use in future RPIPs.
+
+## Specification
+
+### VOPs: What and When?
+This section describes what a VOP is, and when to use the VOP framework.
+
+* A Voted Operational Process (VOP) is a process involving a vote through which an operational decision is made.
+  * Some operational decisions within the Rocket Pool DAOs MAY NOT require a vote, and should not be included under the VOP framework.
+* A VOP MUST NOT be defined to cover decisions that are improvements to the Rocket Pool protocol, Rocket Pool DAOs, or anything that should otherwise require the use of the RPIP process.
+* The dividing line between which decisions should use the RPIP process versus the VOP framework may be unavoidably blurry. The following general recommendations are made:
+  * If a proposal can reasonably predicted to be unique, it is RECOMMENDED to use the RPIP process.
+  * If a proposal is the first of its type or class, and it is not anticipated to repeat frequently or regularly it is RECOMMENDED to use the RPIP process.
+  * If a proposal is not unique or is expected to occur frequently, the VOP framework is RECOMMENDED.
+* The VOP framework SHOULD be used for anything that fits the definition of a voted operational process. 
+  * If an RPIP specifies a process that could reasonably fit within the VOP framework, without adhering to the VOP framework, it MUST include a detailed rationale as to why adhering to the VOP framework is not preferable.
+
+---
+
+### Visual Aid for VOP Concepts
+
+
+![image](../assets/rpip-draft/vops-diagram.png) 
+
+---
+
+### VOP Usage - Definitions
+> The VOP-GMC-MEMBERSELECT definition describes how GMC elections take place.
+
+* A VOP definition contains the elements that make up a particular type of VOP.
+* A VOP definition MUST include the elements defined under the VOP Elements heading of this specification.
+* A VOP definition SHOULD follow the template defined under the VOP Definition Template heading of this specification.
+* If the use of the VOP framework is appropriate, a VOP definition MUST be included in the RPIP in which the process is described.
+
+
+#### VOP Elements
+* Code - A short, unique, human-readable string identifier that can be used to identify instances of this VOP. Should take the rough form of VOP-SUBJECT-ACTION
```

Comment:
Maybe "ID" instead of Code? Code is overloaded. Also works well with the "ID number" nomenclature used later. It's ok as is, fwiw, I just think clearer the other way.

Reply:
Implemented in working copy.